src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/PlayerJoinDelta.java

```diff
@@ -0,0 +1,20 @@
+package ch.uzh.ifi.hase.soprafs22.lobby;
+import ch.uzh.ifi.hase.soprafs22.game.player.IPlayer;
+
+public class PlayerJoinDelta {
```

Comment:
I'd suggest "LobbyDelta" similar to GameDelta

Reply:
Sure, I'll change it